css/tabBar.css

```diff
@@ -38,7 +38,6 @@
 #back-button {
 	opacity: 1;
 	transition: none;
-	font-size: 1.5em;
 	padding: 0 0.6em 0 0.4em;
```

Comment:
The back button isn't aligned correctly anymore:

<img width="250" alt="Screen Shot 2020-07-16 at 11 11 30 AM" src="https://user-images.githubusercontent.com/10314059/87695561-3eeba680-c755-11ea-9aab-8f11e4c2c014.png">

Reply:
This is a bit weird though, it looks like the mac control buttons are misaligned instead of the back button. Or is everything in the menu bar should be a bit offset to the bottom?

Reply:
Hmm, that's a good point. We could change the position of the window controls, and make everything actually centered:

<img width="824" alt="Screen Shot 2020-07-21 at 2 48 15 PM" src="https://user-images.githubusercontent.com/10314059/88099927-7d240400-cb61-11ea-9b5a-e2eb507f9f71.png">

It looks kind of odd to me, but I think that's just because I've spent so much time looking at the old UI, so I'd be fine with doing that if you want.

(To do that, I set `titleBarstyle: 'hidden', trafficLightPosition: {x: 12, y: 19}` on the window, removed the line-height from .tab-item, and repositioned the items using margins).

Reply:
Done :)